app/controllers/api/v1/organizations_controller.rb

```diff
@@ -0,0 +1,33 @@
+# :reek:InstanceVariableAssumption
+class Api::V1::OrganizationsController < ApplicationController
+  before_action :set_organization, only: %i[show update]
+  before_action :authenticate_user!
+
+  def show
+    if @organization && current_user.decorate.organization_member?(@organization.id)
```

Comment:
I don't like the way we decorate user. Maybe it's better to make it once and use everywhere?